src/main/web_src/src/pages/testnorgePage/SearchOptions.tsx

```diff
@@ -18,6 +19,8 @@ export default () => (
 				label="Fødselsdato tom"
 				visHvisAvhuket={false}
 			/>
+			<FormikTextInput name="personinformasjon.alder.fra" label="Fra" visHvisAvhuket={false} />
+			<FormikTextInput name="personinformasjon.alder.til" label="Til" visHvisAvhuket={false} />
```

Comment:
Tror det er lurt å ha "Alder fra" og "Alder til", (evt. "Alder fom" og Alder tom?) for man ser ikke umiddelbart sammenhengen med overskriften 😊